src/encoded/schemas/software.json

```diff
@@ -43,7 +43,14 @@
                     "quantification",
                     "transcript identification",
                     "transcriptome assembly",
-                    "variant annotation"
+                    "variant annotation",
```

Comment:
Can we put the new enums in alphabetical order with the rest of the existing enums? Just makes it easier to navigate through the form.